_source/_posts/2021-11-02-k8s-to-the-cloud-aws.md

```diff
@@ -0,0 +1,391 @@
+---
+layout: blog_post
+title: "Kubernetes To The Cloud With AWS: Deploying a Node.js App to EKS"
+author: victory-nwani
+by: contractor
+communities: [javascript, devops]
+description: "Learn how to deploy a secure Node.js application to Amazon EKS"
+tags: [kubernetes, devops, docker, aws, javascript, okta]
+tweets:
+  - "Want to deploy your #NodeJS application to Amazon EKS? Here is how"
+  - "How to deploy a secure Node.js app to Amazon EKS with few simple steps"
+type: conversion
+image: blog/k8s-to-the-cloud-aws/social.jpg
+---
+
+Since 2013 when it was released to the public, Docker has become an industry-standard tool for development teams to package applications into small executable containers. A year later, Kubernetes was released by Google to manage large amounts of containers, and provide features for High Availability (HA) and auto-scaling.
```

Comment:
```suggestion
Since 2013 when it was released to the public, Docker has become an industry-standard tool for development teams to package applications into small executable containers. A year later, Kubernetes was released by Google to manage large amounts of containers, and provide features for high availability (HA) and auto-scaling.
```
removed capitalization